justfile

```diff
@@ -1,29 +1,32 @@
 alias build := youki-release
 alias youki := youki-dev
 
-ROOT := `git rev-parse --show-toplevel`
+KIND_CLUSTER_NAME := 'youki'
 
 # build
 
 # build all binaries
 build-all: youki-release rust-oci-tests-bin runtimetest
 
 # build youki in dev mode
+[no-cd]
 youki-dev:
-    ./scripts/build.sh -o {{ ROOT }} -c youki
+    ./scripts/build.sh -o . -c youki
```

Comment:
I think using relative path like `.` here can be a footgun down the road. We should use absolute path here, so the intent is explicit. We can either use the old `ROOT` or use `{{ justfile_directory() }}`?